bip-0099.mediawiki

```diff
@@ -350,9 +350,7 @@ worth of blocks).
 
 [3] https://github.com/bitcoin/bips/blob/master/bip-0009.mediawiki
 
-[4] https://github.com/bitcoin/bitcoin/compare/0.11...jtimon:hardfork-timewarp-0.11
-
-[5] Original references:
+[4] Original references:
```

Comment:
It looks like this change would replace footnote 4 with 5, and remove footnote 
5.

Would be good to update the link instead of remove it, if a valid link to the branch exists.

Reply:
It does look good. Let's try to get feedback from the BIP author.